src/github.com/getlantern/flashlight/android/bindings/main_test.go

```diff
@@ -0,0 +1,16 @@
+package flashlight
```

Comment:
A very basic test file that can be used to test the `android` build tag (go test -tags="android") and the exported methods without having to deploy the `.so` everytime.